src/smk/tool/geomark/tool-geomark.js

```diff
@@ -202,6 +202,14 @@ include.module( 'tool-geomark', [
                 }
             } )
 
+            // Override default handling in Identify tool
+            smk.$viewer.handlePick( 1, function ( location ) {
+                if ( !self.active ) {
+                    return;
+                }
+                return true;
```

Comment:
It looks like the Identify tool actually returns a promise that resolves with a value of true. I wonder if we should follow a similar pattern?

Reply:
The identity tool chains a promise onto a function, but here there isn't a function to chain onto, so I'm not sure a promise makes sense. Or am I missing something? Maybe I should return a function that returns true instead of returning true? (That seems to work as well.)

Reply:
Try replacing 'return true;' with 'Promise.resolve(true);'. This should keep the desired behavior while also honoring the existing pattern/method signature.

Reply:
Thanks for this suggestion. It works well. I'll push it soon.